closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -124,7 +134,7 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_DigestDestroyCon
   (JNIEnv *env, jclass thisObj, jlong c) {
 
     OpenSSLMDContext *context = (OpenSSLMDContext*) c;
```

Comment:
Not all of the casts from `jlong` to a pointer go through `intptr_t` - this is one example. Please fix them all.

Reply:
fixed